server/src/user/service.js

```diff
@@ -28,11 +34,14 @@ exports.add = (req, res) => {
   })
 
   newUser.save(newUser)
-    .then(() => res.json(201, {
-      success: true,
-      message: res.__('success.add'),
-      user: newUser
-    }))
+    .then(() => {
+      mailer.send(newUser.email, 'User created', '<h2> Hello the user has been created successfully </h2> <a href="http://localhost:8081/auth/' + newUser.token + '"')
```

Comment:
You didn't use the i18n, even if you wrote the message in en.json

Reply:
something like that?  mailer.send(newUser.email, 'User created', 'mail.activation <a href="http://localhost:8081/auth/' + newUser.token )

Reply:
No, you have to use the res.__('mail.activation', your param here), check my other examples (if you need support, don't hesitate)